python/masonite/requirements.txt

```diff
@@ -0,0 +1,3 @@
+masonite>=2.1.0,<2.2.0
```

Comment:
I find the constraint `masonite==2.1.*` more readable

Reply:
I can add that.